src/hardware/mixer.cpp

```diff
@@ -419,27 +419,51 @@ void MixerChannel::AddSilence()
 	MIXER_UnlockAudioDevice();
 }
 
+static const std::map<FilterState, std::string> filter_state_map = {
+        {FilterState::Off, "disabled"},
+        {FilterState::On, "enabled"},
+        {FilterState::ForcedOn, "enabled (forced)"},
+};
+
+void MixerChannel::SetHighPassFilter(const FilterState state)
+{
+	filters.highpass.state = state;
+
+	auto it = filter_state_map.find(state);
+	if (it != filter_state_map.end()) {
+		auto filter_state = it->second;
```

Comment:
(minor, `const auto` instead of `auto`, because we're not writing to the values)

Reply:
Fixed.